### ISSUE TITLE
Discussions about the user flow.

### ISSUE THREAD

**Issue by romu70** [wallabag/design#1]
For wallabag users using a Mac, you can open the user flow with [Overflow](https://overflow.io/). And for the others, please discuss the user flow based on [this output](https://github.com/wallabag/design/blob/master/output/user%20flow.png).

2018-08-08: I added the very first version of the user flow. Some things are still missing.

**Comment by gnppn** [comment]
Hi!

I read the user flow and on the "Home - articles" page, I don't see any buttons for batch operations (my obsession), like Delete, Mark as read or Tag. It would be really useful to be able to do it from this list with infinite scrolling. :)

Edit: I don't know if this is the right place, but could the document account for a "Resume reading previously open articles" feature?

**Comment by romu70** [comment]
Thanks @gnppn . The user flow is just...a user flow, something to illustrate what could be the reworked general navigation within wallabag. I'll try to post a small video instead of a static image, just to get a better idea.

I'm aware about batch processing, and I won't forget it.

**Comment by gnppn** [comment]
OK, thanks for answering this fast! This flow seems really logical to me. I just have one comment: "Clients and API" is batched with the admin settings. Isn't the user meant to also use this function?

**Comment by romu70** [comment]
Yes of course. But let's says it's the "admin" part of what the user can operate. Let's keep wallabag clean and simple to read articles, and rest, what is not used on regular basis, doesn't need to be always immediatly accessible. You get the point? But I take your comment, and I'll find another term.

**Comment by gnppn** [comment]
Yes I get the logic and I think it's for the best. It's better than the current menu with these settings always displayed.
